app/models/services/tumblr.rb

```diff
@@ -19,9 +19,11 @@ def post(post, url='')
     blogs = user_info["response"]["user"]["blogs"].map { |blog| URI.parse(blog['url']) }
     tumblr_ids = {}
     blogs.each do |blog|
-      resp = client.post("/v2/blog/#{blog.host}/post", body)
-      if resp.code == "201"
-        tumblr_ids[blog.host.to_s] = JSON.parse(resp.body)["response"]["id"]
+      if blog['primary']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.